test/smoke/run-smoke-suite.sh

```diff
@@ -92,24 +92,74 @@ NODE_UUID=$(kubectl exec "${POD}" -c controller -- cat /opt/wallarm/etc/wallarm/
 echo "UUID: ${NODE_UUID}"
 
 echo "Deploying pytest pod ..."
-kubectl run pytest \
-  --env="NODE_BASE_URL=${NODE_BASE_URL}" \
-  --env="NODE_UUID=${NODE_UUID}" \
-  --env="WALLARM_API_HOST=${WALLARM_API_HOST}" \
-  --env="API_CA_VERIFY=${WALLARM_API_CA_VERIFY}" \
-  --env="CLIENT_ID=${CLIENT_ID}" \
-  --env="USER_UUID=${USER_UUID}" \
-  --env="USER_SECRET=${USER_SECRET}" \
-  --env="HOSTNAME_OLD_NODE=${HOSTNAME_OLD_NODE}" \
-  --image="${SMOKE_IMAGE_NAME}:${SMOKE_IMAGE_TAG}" \
-  --image-pull-policy=IfNotPresent \
-  --pod-running-timeout=3m0s \
-  --restart=Never \
-  --overrides='{"apiVersion": "v1", "spec":{"terminationGracePeriodSeconds": 0, "imagePullSecrets": [{"name": "'"${SMOKE_IMAGE_PULL_SECRET_NAME}"'"}]}}' \
-  --command -- sleep infinity
-
-kubectl wait --for=condition=Ready pods --all --timeout=180s
+
+POD_CONFIG=$(cat << EOF
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pytest
+spec:
+  terminationGracePeriodSeconds: 0
+  restartPolicy: Never
+  imagePullSecrets:
+    - name: "${SMOKE_IMAGE_PULL_SECRET_NAME}"
+  containers:
+  - command:
+    - sleep
+    - infinity
+    env:
```

Comment:
can not see where you pass `ALLURE*` vars to the container. How allurectl in contaener knows where to send the report and other settings?

Reply:
this place - https://github.com/wallarm/ingress/blob/e8fb4154ca14bd770c8a0db3e04d79faf8c0e1b2/test/smoke/run-smoke-suite.sh#L165-L173